bin/varnishd/mgt/mgt_jail_linux.c

```diff
@@ -71,7 +73,26 @@ static int vjl_make_subdir(const char *dname, const char *what, struct vsb *vsb)
 }
 
 static int vjl_make_workdir(const char *dname, const char *what, struct vsb *vsb) {
-	return jail_tech_unix.make_workdir(dname, what, vsb);
+	struct statfs info;
+
+	if (jail_tech_unix.make_workdir(dname, what, vsb) != 0)
+		return (1);
+	vjl_master(JAIL_MASTER_FILE);
+	if (statfs(dname, &info) != 0) {
+		if (vsb)
+			VSB_printf(vsb, "Could not stat working directory '%s': %s (%d)\n", dname, VAS_errtxt(errno), errno);
+		else
+			MGT_Complain(C_ERR, "Could not stat working directory '%s': %s (%d)", dname, VAS_errtxt(errno), errno);
+		return (1);
+	}
+	if (info.f_type != TMPFS_MAGIC) {
+		if (vsb != NULL)
+			VSB_printf(vsb, "Working directory not mounted on tmpfs partition\n");
+		else
+			MGT_Complain(C_INFO, "Working directory not mounted on tmpfs partition");
```

Comment:
Should we add a "warning" log level to `MGT_Complain` for this, instead of using info ? (3f877ea925c71c563345fc8ac9940c18c98f892e)

Reply:
I think it could be nice indeed, I was surprised there was no warning log level

Reply:
Bugwash: keep C_INFO